pennylane/qnode.py

```diff
@@ -173,6 +174,10 @@ class QuantumFunctionError(Exception):
     """Exception raised when an illegal operation is defined in a quantum function."""
 
 
+ParDep = namedtuple("ParDep", ["op", "par_idx"])
+"""Represents the dependence of an Operation on a Variable, through its par_idx'th parameter."""
```

Comment:
This reads a bit weirdly. Might be best to be more explicit?
```suggestion
"""Represents the dependence of an Operation on a Variable.

Args:
    op (Operation): an operation containing a :class:`~.Variable` instance
    par_idx (int): the operation argument index of the corresponding :class:`~.Variable`
```